### PR TITLE
complete get comments by article id

### PR DESCRIPTION
**Pull request by amy-baker**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -84,12 +84,13 @@ describe('/api/articles/:id', () => {
     })
 })
 describe('/api/articles', () => {
-    test('responds with an array of article objects with comment_count created from comments.js and body property removed', () => {
+    test('GET 200: responds with an array of article objects with comment_count created from comments.js and body property removed', () => {
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:87]: This test description is me giving me concern that you have not used PSQL appropriately for this end point.

File: db/controller.js [hunk 2]
```diff
@@ -48,4 +48,20 @@ const getAllArticles = (req, res, next) => {
     })
 }
 
-module.exports = {getAllTopics, getAllEndpoints, getArticleById, getAllArticles}
\ No newline at end of file
+const getCommentsById = (req, res, next) => {
+    
+    const { article_id } = req.params;
+    if ( isNaN(article_id)) {
```
Comment (dappernerddesigns) [inline db/controller.js]: PSQL would handle this if you have written the appropriate error handler for a 22P02 error

File: __tests__/app.test.js (hunk 1 above)
Comment (dappernerddesigns) [reply __tests__/app.test.js:87]: I know this isn't the pr you've submitted, and I can see that the query is fine. This description could be something as simple as "Responds with an array of article objects with the required properties"

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
This is very close but missing some error handling in app.js that would make your life easier. You don't need to make a custom error for each time there should be an error code. PSQL returns it's own error code for invalid data types on the code of 22P02. Check the error handling notes on how to do this.

Resubmit when you're ready.

**Review by nc-poppy-m — APPROVED** [review]
Well done on implementing feedback, good to merge!

**Pull request merged**